test/widgets/message_list_test.dart

```diff
@@ -1226,6 +1227,67 @@ void main() {
 
       debugNetworkImageHttpClientProvider = null;
     });
+
+    group('action sheet visual feedback', () {
+      late Message message;
+
+      setUp(() {
+        message = eg.streamMessage();
+      });
+
+      Color? getBackgroundColor(WidgetTester tester) {
+        final decoratedBox = tester.widget<DecoratedBox>(
+          find.descendant(
+            of: find.byType(MessageWithPossibleSender),
+            matching: find.byType(DecoratedBox),
+          ),
+        );
+        return (decoratedBox.decoration as BoxDecoration).color;
+      }
+
+      testWidgets('long-press opens action sheet', (tester) async {
+        await setupMessageListPage(tester, messages: [message]);
+
+        check(getBackgroundColor(tester)).equals(Colors.transparent);
+
+        await tester.longPress(find.byType(MessageWithPossibleSender));
+        await tester.pump();
+
+        final expectedTint = DesignVariables.of(tester.element(find.byType(MessageWithPossibleSender)))
+          .pressedTint;
+
+        check(getBackgroundColor(tester)).equals(expectedTint);
+
+        await tester.pumpAndSettle();
+
+        check(getBackgroundColor(tester)).equals(expectedTint);
+
+        await tester.tapAt(const Offset(0, 0));
+        await tester.pumpAndSettle();
+
+        check(getBackgroundColor(tester)).equals(Colors.transparent);
+      });
+
+      testWidgets('long-press canceled', (tester) async {
+        await setupMessageListPage(tester, messages: [message]);
+
+        check(getBackgroundColor(tester)).equals(Colors.transparent);
+
+        final gesture = await tester.startGesture(tester.getCenter(find.byType(MessageWithPossibleSender)));
+        await tester.pump();
+
+        final expectedTint = DesignVariables.of(tester.element(find.byType(MessageWithPossibleSender)))
+          .pressedTint;
+
+        check(getBackgroundColor(tester)).equals(expectedTint);
+
+        await gesture.moveBy(const Offset(0, 50));
+        await gesture.up();
+        await tester.pumpAndSettle();
```

Comment:
Can we comment out the `gesture.up`? I believe the drag by itself (without lifting a finger) should suffice, and simplifies the storyline.

Also can the `pumpAndSettle` just be a `pump`?